packages/protvista-structure/src/structure-viewer.ts

```diff
@@ -121,33 +121,35 @@ class StructureViewer {
         onHighlightClick([{ position: sequencePosition, chain: chain }]);
       }
     });
-    PluginCommands.Canvas3D.SetSettings(this.plugin, {
-      settings: (props) => {
-        // eslint-disable-next-line no-param-reassign
-        props.renderer.backgroundColor = Color(0xffffff);
-      },
+    const that = this;
```

Comment:
I don't think this is necessary. `this` should work because you are using arrow functions. or am I missing something?

Reply:
You are right, it works, I updated the PR 👍